osh/sh_expr_eval.py

```diff
@@ -80,6 +80,10 @@
 #   (( val = a[key] ))       # osh/sh_expr_eval.py:509 (Id.Arith_LBracket)
 #
 
+# flags for libc functions
+FnmCasefold = 1 << 4
+RegIcase = 1 << 1
```

Comment:
Ditto here, let's create a constant that we can import like this

from libc import REG_ICASE  (or whatever its spelling is)

BTW when I fiddle with `pyext/`, I also use `pyext/libc_test.py`, in case that's helpful